src/oscar/core/loading.py

```diff
@@ -214,11 +214,12 @@ def _get_installed_apps_entry(app_name):
     This does depend on the order of INSTALLED_APPS and will break if
     e.g. 'dashboard.catalogue' comes before 'catalogue' in INSTALLED_APPS.
     """
-    for installed_app in settings.INSTALLED_APPS:
+    for entry in settings.INSTALLED_APPS:
```

Comment:
Can't we use [`get_app_configs()`](https://docs.djangoproject.com/en/2.0/ref/applications/#django.apps.apps.get_app_configs) here instead of re-creating all the configs? Also there might be side effects to initialising configs multiple times.